services/core-api/app/api/projects/project_summary/models/project_summary_authorization.py

```diff
@@ -199,11 +209,12 @@ def create(cls,
             exemption_requested=exemption_requested,
             ams_tracking_number=ams_tracking_number,
             ams_outcome=ams_outcome,
-            ams_status_code=ams_status_code)
+            ams_status_code=ams_status_code,
+            amendment_documents=amendment_documents)
         if add_to_session:
             new_authorization.save(commit=False)
         return new_authorization
-
+# TODO fileupload only when uploading
```

Comment:
I'm not sure I understand the purpose of this TODO.

If it's referencing work that is outside the scope of this ticket, can we make it a bit more descriptive?

Reply:
Not sure if you saw this comment @isuru-aot.  Would be good to have more clarity on what is left TODO here.

Reply:
Yeah, I missed this. Actually this was for my reference and forgot to remove it. The logic for that was moved to project_summary.py

I just removed this.